app/posts/apply-for-teacher-training/2021-07-23-end-of-cycle-2021.md

```diff
@@ -0,0 +1,226 @@
+---
+title: End of cycle 2021
+description: How the application form changes at the end of the 2021 cycle
+date: 2023-07-21
+---
+
+## Timeline
+
+### 7 September 2021 at 6pm
+
+Candidates applying for the first time in the 2020 to 2021 recruitment cycle can no longer submit an application for courses starting in the 2021 to 2022 academic year.
+
+### 21 September 2021 at 6pm
+
+Candidates applying again in the 2020 to 2021 recruitment cycle can no longer submit an application for courses starting in the 2021 to 2022 academic year.
+
+### 5 October 2021 at 9am
+
+Candidates can find courses starting in the 2022 to 2023 academic year ("Find reopens").
+
+### 12 October 2021 at 9am
+
+Candidates can submit an application for courses starting in the 2022 to 2023 academic year ("Apply reopens").
+
+## Candidates are informed about their deadline to apply
+
+### Candidates applying for the fist time
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "candidate-applying-first-time-starts-application-before-deadline.png"
+  },
+  caption: "Deadline banners start to show approximately one month before the deadline to apply."  
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "candidate-applying-for-the-first-time-gets-nudge-with-deadline.png"
+  },
+  caption: "Nudge emails are sent approximately 2 months and then one month before the deadline to apply."   
+}) }}
```

Comment:
Wondering if it makes sense to use the email preview component, rather than screenshots for the emails on this page.